components/faqs/support-for-css-frameworks.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import SectionLayout from '../SectionLayout'
+import CodeBlock from '../CodeBlock'
+
+const sampleTachyonsExample = (`
+const Button = styled.button.attrs({
+  className: 'pa4 ba b--black-80',
+})\`
+	background: red;
+	color: white;
+
+	font-size: 1em;
+	margin: 1em;
+	padding: 0.25em 1em;
+	border: 2px solid palevioletred;
+	border-radius: 3px;
+\`;
+
+/* And use in render as: */
+
+render(
+	<div>
+		<Button>Hello from Tachyons</Button>
+	</div>
+);
+`).trim()
+
+const CSSFrameworks = () => (
+  <SectionLayout title="Support for CSS Frameworks">
```

Comment:
Sure, will do! I used such a heading because nav bar links weren't questions! And clicking on them weren't letting the anchor tags to work. Anyways, I am changing the heading in the section & nav bar. Perhaps, I will also have the links in nav bar changed?